locales/ru-RU/setting.json

```diff
@@ -21,6 +21,14 @@
     "sessionWithName": "Настройки сеанса · {{name}}"
   },
   "llm": {
+    "Anthropic": {
+      "title": "Антропный",
```

Comment:
I think it's not worth translating, but leaving the `Anthropic`. So that it doesn't seem like it's a machine translation

Reply:
I agree

Reply:
@shigomany I agree but this seems to be a general issue to solve in the i18n project. Other providers' titles were mostly translated already

Reply:
I consider to fix it later